Makefile

```
@@ -224,8 +224,12 @@ test-cov-clean:
 test-prometheus: $(PROMTOOL)
 	@./hack/test-prometheus.sh
 
+.PHONY: check-docforge
+check-docforge: $(DOCFORGE)
+	@./hack/check-docforge.sh
+
 .PHONY: verify
-verify: check format test test-integration test-prometheus
+verify: check format test test-integration test-prometheus check-docforge
```

I would propose not to run this in `make verify` and only in `make verify-extended`
```suggestion
verify: check format test test-integration test-prometheus
```

And instead add it to this line https://github.com/gardener/gardener/blob/c57f0e56e7f03b2960722d62fc5640280d31b2b6/.ci/verify#L45